### ISSUE TITLE
python version compatibilty

### ISSUE THREAD

**Issue by indexofrefraction** [dflook/python-minifier#31]
hi, 
i tried this to minify code that needs to run on python 2.7 and python 3.7
sadly print() gets translated to print if minifying is run on 2.7 which breaks compatibilty
exec(...) gets minified to: exec ... also triggering errors on 3.x

it would be nice if there was a compatibilty mode not downgrading all code to 2.x standard
best, index

**Comment by dflook** [comment]
If you run python-minifier using python 3.7, the output will be compatible with 3.7
Do you have any problems running the 3.7 minified code on 2.7?

**Comment by indexofrefraction** [comment]
i'd have to test 3.7 minified code works on 2.7
but for me ideally minification would work from both python versions with results working on both versions, too. 
hence the idea of a compat mode without syntax changes. the code itself works on both versions.

**Comment by dflook** [comment]
Python-minifier relies on the running interpreter for parsing, so it won't have greater forward-compat that python itself.
If there is any code that works on a lower version than it's being minified with but breaks after being minified, that might be worth a compatability mode.

On that note, did you `from __future__ import print_function` in the code where `print()` is transformed to `print`? That would be a bug

**Comment by dflook** [comment]
@rochacbruno Yes. It's been implemented, but must be validated against the release version. If all goes well it will be released today 🤞.

**Comment by rochacbruno** [comment]
Thanks @dflook now dynaconf install test on 3.10 is passing :) 

https://github.com/rochacbruno/dynaconf/runs/3816321083?check_suite_focus=true
